adm/src/main.rs

```diff
@@ -20,6 +20,7 @@ extern crate clap;
 extern crate libc;
 extern crate lmdb_zero;
 extern crate protobuf;
+extern crate sawtooth;
```

Comment:
since this is now 2018 edition, these can be omitted.

Reply:
Fixed